mr/src/test/java/org/apache/mahout/classifier/df/tools/VisualizerTest.java

```diff
@@ -17,9 +17,7 @@
 
 package org.apache.mahout.classifier.df.tools;
 
-import java.util.List;
-import java.util.Random;
-
+import com.google.common.collect.Lists;
```

Comment:
u should commit with the same message as previous one, that way the same PR
is updated

On Sun, Apr 5, 2015 at 2:35 AM, Anand Avati <notifications@github.com>
wrote:

> In
> mr/src/test/java/org/apache/mahout/classifier/df/tools/VisualizerTest.java
> <https://github.com/apache/mahout/pull/105#discussion_r27775259>:
>
> > @@ -17,9 +17,7 @@
> >
> >  package org.apache.mahout.classifier.df.tools;
> >
> > -import java.util.List;
> > -import java.util.Random;
> > -
> > +import com.google.common.collect.Lists;
>
> Not sure why this PR still says 2 commits, there should be 3 now. Can you
> take the third commit manually?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/mahout/pull/105/files#r27775259>.
>